client/src/css/mainApp.css

```diff
@@ -0,0 +1,14 @@
+.trackerCss {
```

Comment:
nit-picky here, but stick with one naming convention for your css, dashes or camelCase, most commonly used is dashes.